lib/identity_cache/query_api.rb

```diff
@@ -120,7 +120,14 @@ def check_association_scope(association_name)
         end
       end
 
-      def record_from_coder(coder) #:nodoc:
+      def record_from_coder(coder)
```

Comment:
I thought the point of the instrumentation was to we don't need to make something like this documented/stable.  It seems like this should still be undocumented.  Maybe instead this should be called `instrumented_record_from_coder` so we don't need to name the method it calls `record_from_coder_internal` which would make this method sound like it isn't internal.